test/verify/check-system-info

```diff
@@ -80,13 +80,14 @@ class TestSystemInfo(testlib.MachineCase):
         m.write("/etc/os-release", os_release)
         m.execute("chattr +i /etc/os-release; (systemctl restart systemd-hostnamed || systemctl restart hostnamed)")
 
+        # set static machine ID, as they have different lengths which disturbs the pixel test
+        m.execute("mv /etc/machine-id /etc/machine-id.orig")
+        m.write("/etc/machine-id", "123456789abcdef123456789abcdef00")
```

Comment:
This test is non-destructive, why don't we use `self.write_file`?

Reply:
It is destructive. I didn't use write_file() because that doesn't have an API to revert to the original file within the test -- and I want the "textual" assertion further down to happen on the actual machine ID. Of course we could also move this around and do that before the pixel check, if you prefer.

Reply:
It's Monday and I mis-autocompleted `skipDistroPackage` to `@nd`.

Reply:
This could maybe also have used the pixel test mock support.